### ISSUE TITLE
several source data on the same timedimension

### ISSUE THREAD

**Issue by gesaleh** [socib/Leaflet.TimeDimension#115]
Hi I have different source with following format json Lat,Long,Value I need to have them on the same map layers and using one control to see time laps 

is it possible with this plugin (BTW it is great)

thanks

**Comment by bielfrontera** [comment]
Hi @gesaleh,
how is your data organised? Do you have several json files, each of them with the data on a certain time? If this is the case, you could create a new class by extension of [TimeDimension.Layer](https://github.com/socib/Leaflet.TimeDimension#ltimedimensionlayer) in order to manage your data. You can follow the instructions on issue #19 or check example 12.

**Comment by gesaleh** [comment]
yes both data are coming through Ajax as 2 different jsons with different times
I'll check example 12
Thanks

**Comment by steven224** [comment]
@gesaleh 
Hi, gesaleh
Did you solved your problem? I am facing same issue...could you share some experience ?

Thanks
